topcoffea/json/data_samples/2017/DoubleEG_D-UL2017.json

```diff
@@ -19,7 +19,6 @@
     "/store/data/Run2017D/DoubleEG/NANOAOD/UL2017_MiniAODv1_NanoAODv2-v1/280000/5BBC2865-F73D-9142-8E5E-EFB9F09A7325.root",
     "/store/data/Run2017D/DoubleEG/NANOAOD/UL2017_MiniAODv1_NanoAODv2-v1/280000/27279C1F-C7D1-0747-A314-5FF573602296.root",
     "/store/data/Run2017D/DoubleEG/NANOAOD/UL2017_MiniAODv1_NanoAODv2-v1/280000/99AA1544-946B-294D-B02F-F7B234700EF7.root",
-    "/store/data/Run2017D/DoubleEG/NANOAOD/UL2017_MiniAODv1_NanoAODv2-v1/280000/15C20D98-A4E8-DD4C-AC5A-319B19FD2C5A.root",
```

Comment:
Was this file bad, or did we lose it on a bad run of `make_jsons.py`?

Reply:
Just asked @Andrew42 and @yanfr0818 about this. Apparently these files were removed by @yanfr0818 at some point, but he is putting them back in.

Reply:
Thanks @yanfr0818 for replacing them!